### PR TITLE
link to Firefox password manager from safety site

### LINKED ISSUE #374: Add link to Firefox password manager from safety article about password managers
@HPaulJohnson and I spoke about this, and he encouraged me to ping @brettgaylor.

We'd like to link from the [password manager page on the safety site](https://advocacy.mozilla.org/en-US/safety/password-manager) to the [password manager page on the Firefox marketing site](https://www.mozilla.org/firefox/features/password-manager/).

Paul asked me to submit some copy. It's attached as an image, and below as text + links. I am particularly interested in the specific phrasing of the anchor text ("password manager in Firefox") and the URL with its params; otherwise, wordsmith away!

Thanks.

> Protect your logins. Use a password manager (like the default [password manager in Firefox](https://www.mozilla.org/firefox/features/password-manager/?utm_source=advocacy.mozilla.org&utm_campaign=safety&utm_medium=referral)) to make creating and remembering passwords easier. Check out [this list from Wired](https://www.wired.com/2016/01/you-need-a-password-manager/) for more options. (Note: We haven't tried them all – see what works for you.)

screen shot 2017-08-07 at 3 49 21 pm

### PR DESCRIPTION
**Pull request by kirinrastogi**
Closes #374 

@hoosteeno I'm a first time contributor to mozilla, is this what you wanted for that link update? I'm also new to webpack.

I also updated the gitignore because npm install would keep making a lock file, and you guys don't have one yet.

Image:
ff-pw-manager

Note: `this list from Wired` is underlined here as it is a link, but not in the image in #374

### CODE REVIEW

**Comment by kirinrastogi** [comment]
cc: @HPaulJohnson @brettgaylor

**Comment by ScottDowne** [comment]
This seems reasonable to me. @HPaulJohnson @brettgaylor I can merge this in if that's cool with either of you.

**Comment by brettgaylor** [comment]
Yes definitely!  Sorry this took so long, and big appreciation for a pull
request :)

File: src/components/safety/gallery-data.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = [
...
     "video": "cIPpBLTC74c",
     "tips": [
-      "Protect your logins. Use a <a href='https://www.wired.com/2016/01/you-need-a-password-manager/'>password manager</a> to make creating and remembering passwords easier. (Note: we haven’t tried them all – see what works for you.)"
+      "Protect your logins. Use a password manager (like the default <a href='https://www.mozilla.org/en-US/firefox/features/password-manager/?utm_campaign=safety&utm_medium=referral&utm_source=advocacy.mozilla.org'>password manager in Firefox</a>) to make creating and remembering passwords easier. Check out <a href=''https://www.wired.com/2016/01/you-need-a-password-manager/>this list from Wired</a> for more options. (Note: we haven’t tried them all – see what works for you.)"
```
Comment (ScottDowne) [inline src/components/safety/gallery-data.js]: I just noticed your second link is a little busted. Not biggie, just looks like a type:

`<a href=''https://www.wired.com/2016/01/you-need-a-password-manager/>this list from Wired</a>`

Looks like the quotes on the `href` are missing the link target.

**Comment by kirinrastogi** [comment]
@ScottDowne thanks, I didn't even notice that. Fixed it

**Comment by ScottDowne** [comment]
It's possible the browser attempted to fix it for you, if you saw it working.

But there is no difference between single and double quotes, if that's what you mean.

I think this looks good now, merging to staging.

**Comment by ScottDowne** [comment]
Should be live on staging momentarily: https://advocacy.mofostaging.net/en-US/safety/password-manager

**Pull request merged**